back/app/services/areas_filtering_service.rb

```diff
@@ -0,0 +1,26 @@
+class AreasFilteringService
+  include Filterer
+
+  add_filter('for_homepage_filter') do |scope, options|
+    params = options[:params]
+    current_user = options[:current_user]
+
+    next scope unless ['true', true, '1'].include? params[:for_homepage_filter]
+
+    homepage_publications =
+      AdminPublicationsService.new.not_draft AdminPublicationPolicy::Scope.new(
+        current_user,
+        AdminPublication
+      ).resolve
```

Comment:
Can homepage publications also include child projects of a folder? If not, then I would also move that logic into the service. If yes, then I misunderstood and I guess then the service is not that useful after all.

Reply:
You're right, my bad. I removed `depth: 0` from the `AdminPublicationsService` def while developing and forgot to put it back.